rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/analysis/ForceAnalyzeResponse.java

```diff
@@ -21,11 +21,12 @@
 
 import java.util.UUID;
 import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
 
 public class ForceAnalyzeResponse {
   UUID analysisId;
 
-  public ForceAnalyzeResponse(UUID analysisId) {
+  public ForceAnalyzeResponse(@Nullable UUID analysisId) {
```

Comment:
This nullable analysis ID comes from a single place which does not look like a strong reason. Maybe we should revisit it. See other comment on AnalysisService.

Reply:
We can revisit yes in the current iteration yes. FYI it's not something that I changed, the analysisId was nullable before, I just fixed the annotation